tools/device_management.py

```diff
@@ -131,10 +134,14 @@ def inner(options):
                 "mbed-cli-auto {}".format(api_key.name),
                 description="cetificate auto-generated by Mbed CLI"
             )
-        LOG.info("Writing developer certificate %s into c file "
+        if getattr(options, 'disable_cert'):
+            LOG.warning("Developer certificate %s not written to "
+                "mbed_cloud_dev_credentials.c", dev_cert_info.name)
```

Comment:
Nit:
> Developer certificate <cert name> not written to "mbed_cloud_dev_credentials.c"

This seems like an odd warning to print. To me it kind of begs the question "should it have written it"?

Perhaps something more intentional sounding?

>Skipping update to developer certificate

I think another option is to not write the warning at all (since you have to explicitly give the argument to skip it).

Reply:
I will change the "warning" to "info", but I think we should indicate that we did skip downloading the developer certificate, just so that they/we can see that the option is working as expected. Also, in lines with the previous definition, instead of 

> Skipping update to developer certificate

we should change this to 

> Skipping download of developer certificate